### PR TITLE
added 3 commands added into the unreal engine blueprint for switching…

### PR DESCRIPTION
**Pull request by Framor81**
… cameras, graphics quality, and sending console commands.

### CODE REVIEW

File: ue5osc/__init__.py [hunk 1]
```diff
@@ -96,6 +96,18 @@ def get_image(self, filename: str) -> bytes:
         image = Image.open(filename)
         return image
 
+    def console(self, message: str) -> None:
+        """Sends messages to be executed in Unreal Engine's console"""
+        self.client.send_message("/console", message)
+
+    def switch_camera(self) -> None:
+        """Switches between camera views. One where the robot is visible or not"""
+        self.client.send_message("/switch/view", 0.0)
```
Comment (anthonyjclark) [inline ue5osc/__init__.py]: Is this a toggle command? Or do we send the mode we want it in?

**Review by anthonyjclark — CHANGES_REQUESTED** [review]

File: ue5osc/__init__.py (hunk 1 above)
Comment (Framor81) [reply ue5osc/__init__.py]: It is a toggle command.

Comment (anthonyjclark) [reply ue5osc/__init__.py]: OK. Can you make that more specific? Maybe like this:

```suggestion
    def toggle_camera_view(self) -> None:
        """Toggles the camera between 1st and 3rd person views."""
        dummy = 0.0
        self.client.send_message("/switch/view", dummy)
```

Comment (anthonyjclark) [reply ue5osc/__init__.py]: You'll also need to merge my changes into yours before I can accept.

**Pull request merged**